src/ovp_pipeline/commands/digest_handler.py

```
@@ -0,0 +1,382 @@
+        return 0
+
+    if args.enqueue_daily and not args.run_now:
+        path = _enqueue_daily(vault)
+        print(f"enqueued: {path}")
+        return 0
+
+    # --run-now (alone, or with --enqueue-daily; same effect)
+    task_path = _enqueue_daily(vault)
+    try:
+        out = dispatch_task(vault, task_path, pack=args.pack)
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)